lightning/src/ln/channel.rs

```diff
@@ -7969,9 +7981,9 @@ where
 	}
 
 	pub fn funding_transaction_signed(
-		&mut self, witnesses: Vec<Witness>,
+		&mut self, funding_txid_signed: Txid, witnesses: Vec<Witness>,
```

Comment:
CI is failing because this parameter is passed in the first commit before later being added here.